tests/features/csvwcommandline.feature

```diff
@@ -72,3 +72,64 @@ Feature: Testing the csvw command group in the CLI
     When the pmdutils command CLI is run with "csvw find-where --negate 'ASK WHERE { ?s a <http://www.w3.org/2004/02/skos/core#ConceptScheme>. }'"
     Then the CLI should succeed
     And the CLI should print "single-measure-bulletin.csv-metadata.json"
+
+  Scenario: The pull command should automatically copy over csv-metadata.json files to ouput_dir.
+    Given the existing test-case files "dcatcli/*"
+    When the pmdutils command CLI is run with "csvw pull single-measure-bulletin.csv-metadata.json"
+    Then the CLI should succeed
+    And the file at "out/single-measure-bulletin.csv-metadata.json" should exist
```

Comment:
Can you assert that the dependent files (i.e. the CSV file plus the code list CSV-Ws) are pulled through too? Those bits are pulled via different bits of code.